### PR TITLE
DB-9141 add DEFAULT column to the sysibm.syscolumns view

### PR DESCRIPTION
**Pull request by yxia92**

### CODE REVIEW

**Comment by yxia92** [comment]
jenkins please test branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/UpgradeScriptForAddDefaultToColumnViewInSYSIBM.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2012 - 2019 Splice Machine, Inc.
```
Comment (changli6) [inline splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/UpgradeScriptForAddDefaultToColumnViewInSYSIBM.java]: 2020

Comment (OlegMazurov) [reply splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/UpgradeScriptForAddDefaultToColumnViewInSYSIBM.java]: Please update the copyright

File: splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java [hunk 2]
```diff
@@ -76,6 +76,7 @@ else if(v1==v2)
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1933), new UpgradeScriptToUpdateViewForSYSCONGLOMERATEINSCHEMAS(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1938), new UpgradeScriptForTriggerWhenClause(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1941), new UpgradeScriptForTableColumnViewInSYSIBM(sdd,tc));
+        scripts.put(new Splice_DD_Version(sdd,3,1,0, 1946), new UpgradeScriptForAddDefaultToColumnViewInSYSIBM(sdd,tc));
```
Comment (dgomezferro) [inline splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java]: We need to update the build number

File: splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/UpgradeScriptForAddDefaultToColumnViewInSYSIBM.java (hunk 1 above)
Comment (yxia92) [reply splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/UpgradeScriptForAddDefaultToColumnViewInSYSIBM.java]: Done, thanks!

File: splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java (hunk 2 above)
Comment (yxia92) [reply splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java]: Updated the build number to 1948 for all 3 branches, thanks!

**Comment by arnaud-splice** [comment]
@yxia92 Can you solve the conflicts and run the tests again?

**Pull request merged**